### ISSUE TITLE
Add command or flag to fetch logs

### ISSUE THREAD

**Issue by deliahu** [cortexlabs/cortex#634]
### Description

Currently, only streaming real-time logs is supported. This command or flag would allow for printing logs since a certain time, or possibly between two times.

**Comment by tharun208** [comment]
I like to work on this

**Comment by deliahu** [comment]
@tharun208 we'd welcome this contribution, thank you! I just updated the description to be more clear, let us know if you have any questions. Since this will touch code that's in the operator, you'll probably want to set up a [development environment](https://www.cortex.dev/contributing/development).

**Comment by deliahu** [comment]
@tharun208 I was just wondering if you are currently working on this? I would just like to know in case other first-time contributors might be interested in implementing it.

**Comment by tharun208** [comment]
@deliahu, no I am not working

**Comment by HowJMay** [comment]
Is it still open?

**Comment by deliahu** [comment]
@HowJMay yes this is still open, and as far as I know, it is not currently being worked on

**Comment by HowJMay** [comment]
Could you give me about one week to examine myself and ensure my ability to implement this requirement?

**Comment by deliahu** [comment]
@HowJMay absolutely; feel free to follow up on this thread if you have any questions
